medusa/databases/main_db.py

```diff
@@ -706,3 +727,163 @@ def create_series_dict():
+        self.connection.action('DROP TABLE IF EXISTS new_tv_episodes;')
+
+        self.connection.action('CREATE TABLE IF NOT EXISTS new_tv_episodes '
+                               '(episode_id INTEGER PRIMARY KEY, showid NUMERIC, indexerid INTEGER, indexer INTEGER, '
+                               'name TEXT, season NUMERIC, episode NUMERIC, description TEXT, airdate NUMERIC, hasnfo NUMERIC, '
+                               'hastbn NUMERIC, status NUMERIC, quality NUMERIC, location TEXT, file_size NUMERIC, release_name TEXT, '
+                               'subtitles TEXT, subtitles_searchcount NUMERIC, subtitles_lastsearch TIMESTAMP, '
+                               'is_proper NUMERIC, scene_season NUMERIC, scene_episode NUMERIC, absolute_number NUMERIC, '
+                               'scene_absolute_number NUMERIC, version NUMERIC DEFAULT -1, release_group TEXT, manually_searched NUMERIC);')
+
+        self.connection.action('INSERT INTO new_tv_episodes (showid, indexerid, indexer, '
+                               'name, season, episode, description, airdate, hasnfo, '
+                               'hastbn, status, quality, location, file_size, release_name, '
+                               'subtitles, subtitles_searchcount, subtitles_lastsearch, '
+                               'is_proper, scene_season, scene_episode, absolute_number, '
+                               'scene_absolute_number, version, release_group, manually_searched) '
+                               'SELECT showid, indexerid, indexer, '
+                               'name, season, episode, description, airdate, hasnfo, '
+                               'hastbn, ep_status, ep_quality, location, file_size, release_name, '
+                               'subtitles, subtitles_searchcount, subtitles_lastsearch, '
+                               'is_proper, scene_season, scene_episode, absolute_number, '
+                               'scene_absolute_number, version, release_group, manually_searched '
+                               'FROM tv_episodes;')
+        self.connection.action("DROP TABLE IF EXISTS tv_episodes;")
+        self.connection.action("ALTER TABLE new_tv_episodes RENAME TO tv_episodes;")
+        self.connection.action("DROP TABLE IF EXISTS new_tv_episodes;")
+
+        log.info(u'Remove the quality from the action field, as this is a composite status')
+        sql_results = self.connection.select("SELECT action FROM history GROUP BY action;")
+        for status in sql_results:
+            split = common.Quality.split_composite_status(status[b'action'])
+            self.connection.action("UPDATE history SET action = ? WHERE action = ?;",
+                                   [split.status, status[b'action']])
+
+        self.inc_minor_version()
+
+
+class ShiftQualities(AddSeparatedStatusQualityFields):
+    """Shift all qualities one place to the left."""
```

Comment:
Same for this class - shoulf be shifting right, not left?
I hope I'm wrong though.